### PR TITLE
Added media

### PR DESCRIPTION
**Pull request by youkarpov**
Have doubts about class names in card for audio class and image class

### CODE REVIEW

**Comment by youkarpov** [comment]
Могу забабахать вращение(могу ли я?)

File: index.html [hunk 1]
```diff
@@ -15,15 +15,22 @@
...
+                </figure>
         </div>
         <div class="card-text">
             <p class="card-text-p">
-                Здесь будет текст карточки!
+                В новом 2023 году ты очевидно думал, что холодно будет в Европе
             </p>
         </div>
         <div class="card-audio">
+            <audio class="Turning-point" src="audio/Directed_by_Robert_B._Weide_-_Theme_meme.mp3">
```
Comment (foxsink) [inline index.html]: Class must match parent node. For example `card-audio-element`.
Don't forget to rename file and move both of folders `images` and `audio` inside `media` folder

File: index.html [hunk 2]
```diff
@@ -15,15 +15,22 @@
 <main class="main-container">
     <div class="card">
         <div class="card-image">
-
+            <figure class="Joke">
+                <img class="Petrosyan" src="images/401x512_0xd42ee430_2359835871452856838.png" alt="Я и без того знаю, какой ты Петросян">
+                <figcaption>
+                    Петросян
+                </figcaption>
+                </figure>
```
Comment (foxsink) [inline index.html]: Every tag must be with class. Please change naming as I mentioned below on `<audio>` tag

```Suggestion
Also rename file to smth easier to understand. For expample `card-image-smthaboutpic`
```

**Review by foxsink — CHANGES_REQUESTED** [review]
Check comments with requested changes

**Review by foxsink — APPROVED** [review]

**Pull request merged**